lib/matplotlib/cbook/__init__.py

```diff
@@ -725,40 +721,43 @@ def remove(self, o):
 
 
 def report_memory(i=0):  # argument may go away
-    """return the memory consumed by process"""
-    from subprocess import Popen, PIPE
+    """Return the memory consumed by the process."""
     pid = os.getpid()
     if sys.platform == 'sunos5':
         try:
-            a2 = Popen(['ps', '-p', '%d' % pid, '-o', 'osz'],
-                       stdout=PIPE).stdout.readlines()
+            a2 = subprocess.Popen(
```

Comment:
that should use check_output...
(or more precisely I think a utility function such as report_memory should probably not support sunos5, unless someone specifically asks for it...)

Reply:
Didn't want to mix the rather technical changes in the PR with other code changes, to kkep the review simpler. But since you asked for it, here it is.